packages/lesswrong/components/comments/CommentsTimeline.tsx

```diff
@@ -80,26 +81,30 @@ const CommentsTimelineFn = ({
       </Typography>
     );
   }
+  
+  const commentNodeProps = {
+    treeOptions: treeOptions,
+    startThreadTruncated: startThreadTruncated,
+    parentCommentId: parentCommentId,
+    parentAnswerId: parentAnswerId,
+    forceSingleLine: forceSingleLine,
+    forceNotSingleLine: forceNotSingleLine,
+    shortform: (treeOptions.post as PostsBase)?.shortform,
```

Comment:
Add a comment explaining why you need to cast.[1]

[1]: https://cea-core.slack.com/archives/C75BWSNBH/p1657891284058449